### DISCUSSION TITLE
Calendar Component new Idea

### DISCUSSION THREAD

**Discussion by VasiHemanth** [shadcn-ui/ui#2617] [Ideas]
I have an idea regarding Calendar component. Right now Calendar component only supports date, I am currently using shadcn in my projects. Calendar component doesn't have an option to make a Month Year selection. If a prop is passed like showMonthYearPicker. It should only shows list of months in that year below is the example from react-datepicker library

<img width="269" alt="showMonthYearPicker" src="https://github.com/shadcn-ui/ui/assets/44475523/3c7d963a-db0c-45b8-abd7-22ea4c1bd687">

**Comment by rodrigo-daher-pimentel** [comment]
+1

Would be really helpful for the UX
